README.md

```diff
@@ -2,10 +2,11 @@
 
 The containerd website at https://containerd.io is built using [Hugo](https://gohugo.io) and published to [Netlify](https://netlify.com).
 
-To develop the site locally in "watch" mode (using Docker):
+To develop the site locally in "watch" mode will require a recent
+release of [`hugo`](https://github.com/gohugoio/hugo/releases):
```

Comment:
/s/will require/first install/
/s/):/). To host your local clone of containerd.io via hugo run:/